lib/cluster/s3_archive_bucket.rb

```diff
@@ -27,6 +27,35 @@ def self.find_or_create(bucket_name)
           ]
         }
       })
+
+      # Do not allow deletions in production
+      if !dev_or_testing_cluster?
+        policy = {
+            "Version" => "2008-10-17",
```

Comment:
Is the version for this specific DCE policy or does it represent something else?